camel-idea-plugin/src/main/java/com/github/cameltooling/idea/annotator/CamelJSonPathAnnotator.java

```diff
@@ -53,7 +49,7 @@ boolean isEnabled() {
     void validateText(@NotNull PsiElement element, @NotNull AnnotationHolder holder, @NotNull String text) {
 
         // only validate if the element is jsonpath element
-        if (getCamelIdeaUtils().isCamelExpression(element, "jsonpath")) {
+        if (camelIdeaUtils.isCamelExpression(element, "jsonpath")) {
```

Comment:
I prefer that we call the method instead of having to deal with fields as it also means a state